src/quantum_annealing/dwave.rs

```diff
@@ -0,0 +1,16 @@
+pub struct DWave {
```

Comment:
We will need to add `mod quantum_annealing` to main.rs (or alternatively to lib.rs). Otherwise the compiler won't see it and doesn't compile anything in this directory.